samples/model/README.md

```diff
@@ -0,0 +1,49 @@
+# Model Sample
+
+This sample demonstrates how to start models/create a model using your own model script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
+
+## Preparing the Model Script
+
+### Get an Image
+
+All SageMaker models are run from within a container with all necessary dependencies and modules pre-installed and with the model scripts referencing the acceptable input and output directories. Sample container images are [available](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-algo-docker-registry-paths.html).
+
+A container image URL and tag looks has the following structure:
+```
+<account number>.dkr.ecr.<region>.amazonaws.com/<image name>:<tag>
+```
+
+### Updating the Model Specification
+
+In the `my-model.yaml` file, modify the placeholder values with those associated with your account and model.
+
+# Submitting your Model
+
+## Create your Model
+To submit your prepared model specification, apply the specification to your EKS cluster as such:
+```
+$ kubectl apply -f my-model.yaml
+models.sagemaker.services.k8s.aws.amazon.com/my-model created
+```
+
+## List your Model
+To monitor the model status, you can use the following command:
+```
+$ kubectl get models.sagemaker.services.k8s.aws my-model
+```
+
+## Describe your Model
+To monitor the model in-depth once it has started, you can see the full status and any additional errors with the following command:
```

Comment:
can we reword this line a little bit to change the words `monitor` and `in-depth`?

one possible line - 
To get more details about the resource once it's submitted, like checking the status, errors or parameters of the resource . . .